src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,51 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+
+        Manufacturer bmw = new Manufacturer(null, "BMW", "Germany");
+        Manufacturer mustang = new Manufacturer(null, "Mustang", "USA");
+        Manufacturer audi = new Manufacturer(null, "Audi", "Germany");
```

Comment:
```suggestion
        Manufacturer bmw = new Manufacturer("BMW", "Germany");
        Manufacturer mustang = new Manufacturer("Mustang", "USA");
        Manufacturer audi = new Manufacturer("Audi", "Germany");
```
instead of passing nulls to the constructor, create constructor with fewer parameters
or just use setters and default constructor